server/datastore/mysql/maintained_apps.go

```diff
@@ -52,3 +52,51 @@ ON DUPLICATE KEY UPDATE
 		return ctxerr.Wrap(ctx, err, "upsert maintained app")
 	})
 }
+
+func (ds *Datastore) ListAvailableFleetMaintainedApps(ctx context.Context, teamID uint, opt fleet.ListOptions) ([]fleet.FleetMaintainedAppAvailable, *fleet.PaginationMetadata, error) {
+	stmt := `
+SELECT
+	fla.id,
+	fla.name,
+	fla.version,
+	fla.platform
+FROM
+	fleet_library_apps fla
+WHERE NOT EXISTS (
```

Comment:
nice! so this was better than the subquery + where?

Reply:
I actually never ended up testing the performance, although I assume this solution will work well.